olm-catalog/serverless-operator/index/configs/index.yaml

```diff
@@ -3,37 +3,37 @@ schema: olm.channel
 name: stable
 package: serverless-operator
 entries:
-  - name: "serverless-operator.v1.30.0"
   - name: "serverless-operator.v1.31.0"
-    replaces: "serverless-operator.v1.30.0"
-    skipRange: ">=1.30.0 <1.31.0"
-  - name: "serverless-operator.v1.32.0"
+  - name: "serverless-operator.v1.31.0"
+    replaces: "serverless-operator.v1.31.0"
+    skipRange: ">=1.31.0 <1.31.0"
+  - name: "serverless-operator.v1.32.1"
     replaces: "serverless-operator.v1.31.0"
-    skipRange: ">=1.31.0 <1.32.0"
+    skipRange: ">=1.31.0 <1.32.1"
 ---
 schema: olm.channel
 name: stable-1.32
 package: serverless-operator
 entries:
-  - name: "serverless-operator.v1.30.0"
   - name: "serverless-operator.v1.31.0"
-    replaces: "serverless-operator.v1.30.0"
-    skipRange: ">=1.30.0 <1.31.0"
-  - name: "serverless-operator.v1.32.0"
+  - name: "serverless-operator.v1.31.0"
+    replaces: "serverless-operator.v1.31.0"
+    skipRange: ">=1.31.0 <1.31.0"
+  - name: "serverless-operator.v1.32.1"
     replaces: "serverless-operator.v1.31.0"
-    skipRange: ">=1.31.0 <1.32.0"
+    skipRange: ">=1.31.0 <1.32.1"
 ---
 schema: olm.channel
 name: stable-1.31
 package: serverless-operator
 entries:
-  - name: "serverless-operator.v1.30.0"
   - name: "serverless-operator.v1.31.0"
-    replaces: "serverless-operator.v1.30.0"
-    skipRange: ">=1.30.0 <1.31.0"
+  - name: "serverless-operator.v1.31.0"
+    replaces: "serverless-operator.v1.31.0"
+    skipRange: ">=1.31.0 <1.31.0"
 ---
 schema: olm.channel
 name: stable-1.30
 package: serverless-operator
 entries:
-  - name: "serverless-operator.v1.30.0"
+  - name: "serverless-operator.v1.31.0"
```

Comment:
This one doesn't seem right, @mgencur @Kaustubh-pande do you know what I'm doing wrong here ?

Reply:
We need stable 1.30? already EOL?
I think you are adding serverless-operator-1.31.0 in the stable 1.30 channel

Reply:
I think we leave current - 2 for the upgrade test regardless of the EOL date

> I think you are adding serverless-operator-1.31.0 in the stable 1.30 channel

yes, but this is generated and I'm not sure what's wrong in the project.yaml that causes it to generate this one

Reply:
Fixing it here: https://github.com/openshift-knative/serverless-operator/pull/2583 , The commit can be cherry-picked on release-1.32 if desired.